tests/suite/test_app_protect_grpc.py

```diff
@@ -0,0 +1,171 @@
+import subprocess
+import pytest
+from python_hosts import Hosts, HostsEntry
+from settings import TEST_DATA, DEPLOYMENTS
+from suite.custom_resources_utils import (
+    create_ap_logconf_from_yaml,
+    create_ap_policy_from_yaml,
+    delete_ap_policy,
+    delete_ap_logconf,
+)
+from suite.resources_utils import (
+    wait_before_test,
+    create_example_app,
+    wait_until_all_pods_are_ready,
+    create_items_from_yaml,
+    delete_items_from_yaml,
+    delete_common_app,
+    replace_configmap_from_yaml,
+    create_ingress_with_ap_annotations,
+    wait_before_test,
+    get_file_contents,
+)
+from suite.yaml_utils import get_first_ingress_host_from_yaml
+
+my_hosts = Hosts()
```

Comment:
Are these global? Generaly that's not a great idea, i.e.  `my_hosts` can't be declared in the same way in another file. I don't know Python though, so I could be wrong.